jasmine-standalone-3.5.0/src/tenPinApp.js

```diff
@@ -0,0 +1,43 @@
+function Bowling () {
+this.player_score = 0;
+this.strike = false;
+this.spare = false;
+this.frame = 0;
+this.bonus = false;
+}
+
+Bowling.prototype.calcPoints = function (r1,r2){
```

Comment:
Did you plan model/beforehand to make sure that you understand the rules correctly? Did you go and look at other examples?

Reply:
It feels like this should be called differently. Maybe addRoll or something.